### ISSUE TITLE
Won't install alongside puppetlabs-mysql

### ISSUE THREAD

**Issue by threepistons** [jenkinsci/puppet-jenkins#623]
## What I do
Log in as root and run ```puppet module install rtyler-jenkins```

## What I expect
rtyler-jenkins to install.

## What I get
```
[root@hg-puppet ~]# puppet module install rtyler-jenkins
Notice: Preparing to install into /etc/puppetlabs/code/environments/production/modules ...
Notice: Downloading from https://forgeapi.puppetlabs.com ...
Error: Could not install module 'rtyler-jenkins' (latest)
  Dependency 'nanliu-staging' (v1.0.3) would overwrite /etc/puppetlabs/code/environments/production/modules/staging
    Currently, 'puppet-staging' (v1.0.6) is installed to that directory
    Use `puppet module install --ignore-dependencies` to install only this module
```

## Why it happens
puppetlabs-mysql depends upon puppet-staging and puppet-staging conflicts with nanliu-staging.

**Comment by jhoblitt** [comment]
Care to make a PR to update the module namespace?

**Comment by threepistons** [comment]
This has been fixed in 1.7.0.  The module now installs and deploys.

**Comment by threepistons** [comment]
@jhoblitt I'd love to, but I don't currently have permission to publish patches written at work.
